### PR TITLE
Master appraisal skills report sgv

### PR DESCRIPTION
**Pull request by sofiagvaladze**
task - 2687370

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/controllers/main.py [hunk 1]
```diff
@@ -705,6 +705,7 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
 
     def write_header(self):
         # Write main header
+        i = 0
```
Comment (xmo-odoo) [inline addons/web/controllers/main.py:708]: Doesn't seem to belong in this PR.

Also seems like it would be simpler to just not do anything if `self.field_names` is empty? Setting the first column to 30 chars when we have written a grand total of 0 columns doesn't seem very useful.

Comment (sofiagvaladze) [reply addons/web/controllers/main.py]: Hello @xmo-odoo :wave: 

Indeed, I made a separate PR for it, taking into account your advice - https://github.com/odoo/odoo/pull/84191

Would you mind to take a look? :) 

P.S. sorry for the late follow up

**Comment by kba-odoo** [comment]
robodoo r+
